From: Florian Fainelli <f.fainelli@gmail.com>
Date: Sun, 22 Jan 2017 21:17:32 -0800
Subject: net: phy: Fix typo for MDIO module boilerplate comment
Patch-mainline: v4.11-rc1
Git-commit: b70f43a16182c7dbc0779bc5bd9f621711f13eb3
References: bsc#1026030 FATE#321670

The module boilerplate macro is named mdio_module_driver and not
module_mdio_driver, fix that.

Fixes: a9049e0c513c ("mdio: Add support for mdio drivers.")
Signed-off-by: Florian Fainelli <f.fainelli@gmail.com>
Reviewed-by: Andrew Lunn <andrew@lunn.ch>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 include/linux/mdio.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/mdio.h
+++ b/include/linux/mdio.h
@@ -233,7 +233,7 @@ bool mdiobus_is_registered_device(struct
 struct phy_device *mdiobus_get_phy(struct mii_bus *bus, int addr);
 
 /**
- * module_mdio_driver() - Helper macro for registering mdio drivers
+ * mdio_module_driver() - Helper macro for registering mdio drivers
  *
  * Helper macro for MDIO drivers which do not do anything special in module
  * init/exit. Each module may only use this macro once, and calling it
